eternaleconomy-core/src/main/java/com/eternalcode/economy/command/cooldown/CommandCooldownConfig.java

```diff
@@ -0,0 +1,16 @@
+package com.eternalcode.economy.command.cooldown;
+
+import com.eternalcode.multification.notice.Notice;
+import eu.okaeri.configs.OkaeriConfig;
+
+import java.time.Duration;
+
+public class CommandCooldownConfig extends OkaeriConfig {
+    public Duration duration = Duration.ofSeconds(5);
+    public String bypassPermission = "eternaleconomy.player.pay.bypass";
+    public Notice message = Notice.builder()
+        .chat("<b><gradient:#00FFA2:#34AE00>ECONOMY</gradient></b> <dark_gray>➤</dark_gray> "
```

Comment:
maybe make a template for that to use in other notifications?? like static preset

Reply:
unrelated. #44